examples/stats.php

```diff
@@ -4,17 +4,14 @@
 
 use Amp\Beanstalk\BeanstalkClient;
 use Amp\Beanstalk\Stats\System;
-use Amp\Loop;
 
-Loop::run(function () {
-    $beanstalk = new BeanstalkClient("tcp://127.0.0.1:11300");
+$beanstalk = new BeanstalkClient('tcp://127.0.0.1:11300');
 
-    /**
-     * @var System $systemStats
-     */
-    $systemStats = yield $beanstalk->getSystemStats();
-    echo "Active connections: {$systemStats->currentConnections}\n";
-    echo "Jobs ready: {$systemStats->currentJobsReady}\n";
+/**
+ * @var System $systemStats
+ */
```

Comment:
We shouldn't need such comments any longer, because we use proper types instead of generics inside promises.